lib/index.js

```diff
@@ -57,42 +57,63 @@ bedrock.events.on('bedrock.configure', function applyConfig() {
   }
 });
 
+function handleConfigLoadError(e, {configType}) {
+  // Only rethrow raw error in development to avoid leaking sensitive info.
+  if(process.env.NODE_ENV == 'development') {
+    throw e;
+  }
+  logger.error(
+    `Error loading "${configType}" configuration from environment`,
+  );
```

Comment:
Style + no additional param:

```suggestion
  logger.error(
    `Error loading "${configType}" configuration from environment`);
```